doc/api/dgram.md

```diff
@@ -425,9 +425,7 @@ s.bind(1234, () => {
 });
 ```
 
-## `dgram` module functions
```

Comment:
why? you just removed this header, and bumped out the level, so now all the `dgram.*` functions are documented as methods of the dgram.Socket class.